packages/client/hmi-client/src/components/workflow/tera-simulate-chart.vue

```diff
@@ -228,8 +231,47 @@ const renderGraph = () => {
 					borderWidth: lineWidthArray.value[runIdx]
 				};
 				datasets.push(dataset);
-			})
-	);
+			});
+
+		if (props.initialData) {
+			let v = variable;
+			// get the dataset variable from the mapping of model variable to dataset variable if there's a mapping
+			if (props.mapping) {
+				if (!(props.mapping.length === 1 && Object.values(props.mapping[0]).some((val) => !val))) {
+					const varMap = props.mapping.find((m) => m.modelVariable === variable);
+					if (varMap) {
+						v = varMap.datasetVariable;
+					}
+				}
+			}
+
+			// get  the index of the timestamp column
+			const tIndex = props.initialData.headers.indexOf('timestamp');
+
+			// get the index of the variable column
+			let colIdx: number;
+			for (let i = 0; i < props.initialData.headers.length; i++) {
+				if (props.initialData.headers[i].trim() === v.trim()) {
+					colIdx = i;
+					break;
+				}
+			}
+
+			const dataset = {
+				// ignore the first row, it's the header
+				data: props.initialData.csv.slice(1).map((datum: string[]) => ({
+					x: +datum[tIndex].trim(),
+					y: +datum[colIdx].trim()
+				})),
+				label: `${variable} - dataset`,
+				fill: false,
+				borderColor: '#000000',
+				borderWidth: 3
+			};
```

Comment:
It is difficult to tell the lines apart, can we try something like a dashed style for training-data? 

Also feel free to post previews to slack channel to get folks' thoughts and comments

Reply:
Yeah I agree that it's difficult to distinguish the lines. I'll give dashed lines a shot.